handler/src/main/java/io/netty/handler/ssl/OpenSslKeyMaterialManager.java

```diff
@@ -116,12 +117,12 @@ private void setKeyMaterial(long ssl, String alias) throws SSLException {
         }
     }
 
-    protected String chooseClientAlias(@SuppressWarnings("unused") OpenSslEngine engine,
+    protected String chooseClientAlias(@SuppressWarnings("unused") ReferenceCountedOpenSslEngine engine,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "engine". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)